MetriCam2.Controls/CameraSettingsControl.cs

```diff
@@ -650,5 +695,11 @@ public static Point2i ResolutionToPoint2i(string s)
             string[] stringValue = s.Split('x');
             return new Point2i(int.Parse(stringValue[0]), int.Parse(stringValue[1]));
         }
+
+        private void ShowMessageBox()
+        {
+            InitConfigurationParameters(this.Camera);
+            MessageBox.Show("This Configuration is not supported by this camera");
```

Comment:
MetriCam 2 uses the terms "Parameters" and "Settings" - maybe not super consistently - but never "Configuration".

This also applies to the name of the new exception type.